server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoNetworkError/index.ts

```diff
@@ -20,12 +20,45 @@ class MongoNetworkTransientTransactionError extends BaseMongoExceptionRetryRule
 	}
 }
 
+class MongoNetworkConnectionClosedError extends BaseMongoExceptionRetryRule {
+	protected defaultRetryDecision: boolean = true;
+
+	constructor(retryRuleOverride: Map<string, boolean>) {
+		super("MongoNetworkConnectionClosedError", retryRuleOverride);
```

Comment:
Is this a new exception from the new driver? I suspect it was the `MongoNetworkTransientTransactionError`?

Reply:
Yes it seems similar to MongoNetworkTransientTransactionError from the old sdk, but in a different format.